### DISCUSSION TITLE
漢化包使用後的疑問

### DISCUSSION THREAD

**Discussion by Soren-1994** [GpointChen/FFXIVChnTextPatch-GP#465] [General]
請問一下使用漢化包後,會被其他玩家發現嗎?例如跳成就和稱號顯示。因為有聽說過把物品發去公共頻道後,其他玩家看到的是漢化後的物品名稱。

**Comment by GpointChen** [comment]
說話/打字的會

成就和稱號顯示不會

**Comment by Soren-1994** [comment]
明白!感謝回覆!

**Comment by kalispp** [comment]
補充，貼物品資訊或裝備資訊，不管甚麼頻道，對方看到都是漢化後的名稱喔，建議不要隨意貼
